Chaincase.UI/Components/FeeModal.razor

```diff
@@ -0,0 +1,99 @@
+﻿@using WalletWasabi.Exceptions
+@using WalletWasabi.Helpers
+@inherits ReactiveComponentBase<FeeViewModel>
+
+<IonModal @bind-Visible="@Visible">
+    <ion-header>
+        <ion-toolbar>
+            <ion-title>Select Fee</ion-title>
+            <ion-buttons slot="start">
+                <IonButton @onclick="CloseModal">Back</IonButton>
+            </ion-buttons>
+        </ion-toolbar>
+    </ion-header>
+
+    <ion-content class="ion-padding">
+        <p>
+            Bitcoin miners prioritize transactions that pay them the highest fee.
+            Only a limited number can be mined every 10 minute block.
+            These network fees are typically the least expensive on weekends and holidays and most costly when there new public interest or price changes.
+            The arrival estimate is based on current network status and is not a guarantee.
+        </p>
+        <p>
+            Fee:
+            <ion-badge class="ion-text-center">~@ViewModel.SendViewModel.FeeRate</ion-badge>
+            @FeeTargetTimeString
+        </p>
+        <IonRange min="@(-1 * ViewModel.SendViewModel.MaximumFeeTarget)"
+                  max="@(-1 * ViewModel.SendViewModel.MinimumFeeTarget)"
```

Comment:
Are these suppose to be negative values?

Reply:
yes. MinimiumFeeTarget=2 is the minimum numbe of blocks and maximum fee. Consumers are used to least expensive on the left most expensive on the right